packages/daemon/src/rpc/methods/vault.js

```diff
@@ -18,6 +18,7 @@ export const create = {
   handler: async (ctx: ClientContext, params: VaultParams) => {
     try {
       await ctx.mutations.createVault(params.path, Buffer.from(params.password))
+      ctx.io.eth.setup()
```

Comment:
This is already handled by `ctx.mutations.createVault()`, I moved the logic there